synapse/storage/database.py

```diff
@@ -107,20 +99,25 @@ def make_conn(db_config: DatabaseConnectionConfig, engine):
     return db_conn
 
 
-class LoggingTransaction(object):
+# the type of entry which goes on our after_callbacks and exception_callbacks lists
+# use of a string here because python 3.5.2 doesn't support Callable([...]).
```

Comment:
Hmm, I think I sort of understand this comment, but not really sure and I had to read it a few times to get there, could you maybe rephrase it?